packages/orchestration/src/examples/stakeAtom.contract.js

```diff
@@ -63,11 +63,15 @@ export const start = async (zcf, privateArgs, baggage) => {
     const accountAddress = await E(account).getAddress();
     trace('account address', accountAddress);
     const { holder, invitationMakers } = makeStakingAccountKit(
-      account,
-      storageNode,
       accountAddress,
-      icqConnection,
       bondDenom,
+      {
+        account,
+        storageNode,
+        icqConnection,
+        // @ts-expect-error only for undelegate, which we do not use
+        timer: harden({}),
```

Comment:
should it be optional then?

Reply:
Seems worthwhile to include `ChainTimerService` in `start-stakeAtom.js`? `makeAccount` in `StakeAtom` returns a `StakingAccountKit`, so it does seem to be used.

Reply:
oops... missed this one; tending to it now...